src/ai/backend/console/server.py

```diff
@@ -89,10 +89,15 @@
     request_path = request.match_info['path']
     file_path = (static_path / request_path).resolve()
     config = request.app['config']
+    if 'force-endpoint-protocol' in config['service'] and \
+            config['service']['force-endpoint-protocol'] in ['http', 'https']:
+        scheme = config['service']['force-endpoint-protocol']
+    else:
+        scheme = request.scheme
```

Comment:
I would write like:
```suggestion
    scheme = config['service'].get('force-endpoint-protocol')
    if scheme is None:
        scheme = request.scheme
```
and move the config validation to the server initialization part.